### DISCUSSION TITLE
URGENT - Errors after upgradin to 6.29.0 from 6.6.2

### DISCUSSION THREAD

**Discussion by usmansagri** [sequelize/sequelize#15723] [Help & Questions] [answered]
Hi Guys,

Good day!

Due to critical vulnerability(CVE-2023-22578), We are trying to upgrade sequelize to latest available version and getting errors saying 
1. TS1005: '(' expected. 
2. TS1128: Declaration or statement expected.
3. TS1005: ',' expected.

and more.. 

This occures in files like model.d.ts, bulk-record-error.d.ts, validation-error.d.ts, sequelize.d.ts

I am not sure if we need to make any additional changes, Kindly guide how to fix them.

**Comment by ephys** [comment, marked as answer]
It's very likely that your issue comes from the use of an outdated TypeScript version

See https://sequelize.org/releases/ for details on which version of TypeScript Sequelize supports
